### ISSUE TITLE
Mattermost screen reader failure after recent upgrade

### ISSUE THREAD

**Issue by JC-Boeing** [mattermost/mattermost-server#14621]
#### Summary
Screen reader only reads dates, but not the messages, or who sent them.

#### Steps to reproduce
Currently using the following screen reader software: JAWS for Windows 2019 by Freedom Scientific. To reproduce the issue, into Mattermost Web App with Firefox 68.7 and try to read messages, with JAWS 2019, or the latest NVDA by NV Access. 

#### Expected behavior
Expect to be able to read the date a message was sent, who sent it, and the message. This functionality worked in earlier Mattermost versions 5.18 and 5.14 were documented to support accessibility and screen readers.

#### Observed behavior (that appears unintentional) 
Only dates are read, but not the messages or who sent them. There are no errors or screenshots that can be produced in this situation.

**Comment by amyblais** [comment]
@JC-Boeing Thank you for the report,

Are you on Mattermost server v5.23?

**Comment by JC-Boeing** [comment]
@amyblais 4.4.1

**Comment by amyblais** [comment]
Thank you for the report, I opened a bug ticket here https://mattermost.atlassian.net/browse/MM-25471.

**Comment by bbodenmiller** [comment]
@amyblais it's not working on http://community.mattermost.com/. Sounds like it doesn't work in Firefox or latest Desktop app. I'm working with the same person @JC-Boeing reported about.

**Comment by amyblais** [comment]
@bbodenmiller Thanks, a bug ticket for this has been opened here: https://mattermost.atlassian.net/browse/MM-25471.
